src/index.html

```diff
@@ -7,8 +7,63 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class ="link__logo" href="https://www.moyo.ua/">
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="moyo-header-left-logo"
+        >
+      </a>
+      <nav class="navbar">
+        <ul class="nav__list">
+          <li class="nav__item item1">
+            <a class = "nav__link is-active" href="#apple">APPLE</a>
+          </li>
+
+          <li class="nav__item item2">
+            <a class = "nav__link" href="#samsung">SAMSUNG</a>
+          </li>
```

Comment:
1. no needs to use selectors such as `item1`, `item2` etc
2. use lowercased value in html and change it at css
```suggestion
          <li class="nav__item">
            <a class = "nav__link is-active" href="#apple">apple</a>
          </li>

          <li class="nav__item">
            <a class = "nav__link" href="#samsung">samsung</a>
          </li>
```